README.md

```diff
@@ -166,17 +188,28 @@ It can be useful to run the Membrane in a 'service only' mode, where the cloud A
 
 The Membrane project source code structure is outlined below:
 
-Directory                   | Package    | Description
----------                   |----------- |------------
-`/interfaces/nitric/v1`     | `v1`       | protoc generated GRPC services code 
-`/pkg/adapters/grpc`        | `grpc`     | GRPC service to SDK adaptors 
-`/pkg/membrane`             | `membrane` | membrane application
-`/pkg/plugins/...`          | `...`      | Cloud service SDK plugins 
-`/pkg/providers/...`        | `main`     | Cloud provider main application and plugin injection 
-`/pkg/sdk`                  | `sdk`      | SDK service interfaces 
-`/pkg/triggers`             | `triggers` | provides Nitric event triggers
-`/pkg/utils`                | `utils`    | provides utility functions
-`/pkg/worker`               | `worker`   | Membrane workers representing function/service connections
-`/tests/mocks/...`          | `...`      | Cloud service SDK mocks 
-`/tests/plugins/...`        | `...`      | Plugin services integration test suites
-`/tools`                    | `tools`    | include for 3rd party build tools
\ No newline at end of file
+| Directory               | Package    | Description                                                |
+| ----------------------- | ---------- | ---------------------------------------------------------- |
+| `/interfaces/nitric/v1` | `v1`       | protoc generated GRPC services code                        |
+| `/pkg/adapters/grpc`    | `grpc`     | GRPC service to SDK adaptors                               |
+| `/pkg/membrane`         | `membrane` | membrane application                                       |
+| `/pkg/plugins/...`      | `...`      | Cloud service SDK plugins                                  |
+| `/pkg/providers/...`    | `main`     | Cloud provider main application and plugin injection       |
+| `/pkg/sdk`              | `sdk`      | SDK service interfaces                                     |
+| `/pkg/triggers`         | `triggers` | provides Nitric event triggers                             |
+| `/pkg/utils`            | `utils`    | provides utility functions                                 |
+| `/pkg/worker`           | `worker`   | Membrane workers representing function/service connections |
+| `/tests/mocks/...`      | `...`      | Cloud service SDK mocks                                    |
+| `/tests/plugins/...`    | `...`      | Plugin services integration test suites                    |
+| `/tools`                | `tools`    | include for 3rd party build tools                          |
+
+## Ideas for using Nitric
+
+Examples of projects built with Nitric:
+
+- Identity verification APIs
+- FinTech APIs with complex business rules
+- Upgrade Express.js or koa backend projects
```

Comment:
It's not so much an upgrade as a move to serverless and distributed architecture (functions/microservices). Express and Koa are still awesome for the right use cases. I think we should be more clear about that here.

Reply:
Agreed, changed as suggested.